dask/array/slicing.py

```diff
@@ -13,6 +13,7 @@
 from ..base import is_dask_collection, tokenize
 from ..highlevelgraph import HighLevelGraph
 from ..utils import is_arraylike
+from .utils import getitem
```

Comment:
Can you try putting this import at the bottom of this file and see if that fixes things?

Reply:
No, it did not work. Although I noticed that if I put the function somewhere else like `dask/utils.py` instead of `dask/array/utils.py` it does work, but this is not the ideal place for that function to be since it's mainly related to `dask/array`. I'll keep digging into it.

Reply:
I looked at imports and it looks like dask/array/utils.py depends on dask/array/core.py (it imports core.Array) and dask/array/core.py depends on dask/array/slicing.py, so there is a circular import.

So you could put this in core.py, or there is also a module, chunk.py which I think was created for this purpose.

Reply:
`core.Array` is only used in 4 places in `array/utils` for isinstance checks. Another options is to just move the `from .core import Array` inside each of those functions that needs it.

Reply:
It seems the approach mention by @Illviljan was the one taken in `array/slicing.py` where the import is within the scope of the function. I'll go that route, and if we think this is not optimal we'll modify it.